components/ui/FooterNew.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { FC } from "react";
```

Comment:
Is it possible to swap out our current footer with this component?

Reply:
Yes, but it won't respond correctly to breakpoints since the right/left drawers are still there. Might be best to wait for now until they are removed.